tests/python/unittest/test_operator.py

```diff
@@ -955,6 +955,40 @@ def test_convolution_grouping():
     for arr1, arr2 in zip(exe1.outputs + exe1.grad_arrays, exe2.outputs + exe2.grad_arrays):
         np.testing.assert_allclose(arr1.asnumpy(), arr2.asnumpy(), rtol=1e-3, atol=1e-4)
 
+def test_depthwise_convolution():
```

Comment:
Actually, this is fine. please reduce 224 to something like 32 or 64 and test for 2 or 3 more configs. Like differetn num_base and kernal_size/pad/stride